pack/tsk/tskp.json

```
@@ -29,7 +29,98 @@
         "traits": "Flaw.",
         "type_code": "treachery"
     },
-
+    {
+        "back_flavor": "After the third or fourth mutilated body showed up during his time at St. Mary’s Hospital in Arkham, Vincent Lee began to suspect that these were not mere wild animal attacks. When a young man died screaming in his sleep from a burst heart, Vincent knew for a fact that dark times were ahead. Taking a leave of absence from his position at the hospital, he has set out to ease the suffering of those afflicted by the spreading chaos, hoping that the trail of bodies will lead him to the answers he craves.",
+        "back_text": "<b>Deck Size</b>: 30.\n<b>Deckbuilding Options</b>: Seeker cards ([seeker]) level 0-3, Neutral cards level 0-5, cards that \"heal damage\" level 0-5, up to 15 other Guardian and/or Survivor cards level 0-1 ([guardian] and/or [survivor]).\n<b>Deckbuilding Requirements</b> (do not count toward deck size): Bonesaw, 1 [per_investigator] copies of On the Mend, Wounded Bystander, 1 random basic weakness.\n<b>Additional Setup</b>: You begin each game with 1 [per_investigator] copies of On the Mend set aside, out of play.",
+        "code": "09004",
+        "deck_limit": 1,
+        "deck_options": [
+            { "faction": ["seeker"], "level": { "min": 0, "max": 3 } },
+            { "faction": ["neutral"], "level": { "min": 0, "max": 5 } },
+            {
+                "text": [
+                    "[Hh]eals? (that much )?(\\d+ horror (and|or) )?(\\d+|all|(X total) )?damage"
```

I think we should consider just using the flipped Carolyn.  As far as I can tell, this grabs all the cards:

```
First Aid (0)
Medical Texts (0)
First Aid (0)
Medical Texts (0)
First Aid (3)
Emergency Aid (0)
Painkillers (0)
Strange Solution - Restorative Concoction (4)
Inspiring Presence (0)
First Aid (3)
Cheat Death (5)
Second Wind (0)
Thermos (0)
Deny Existence (5)
Jessica Hyde - Wrong Place, Wrong Time (1)
Healing Words (0)
The Black Cat - A Liar, or a Prophet, or Both (5)
Spiritual Resolve (5)
Manipulate Destiny (2)
Spirit of Humanity (2)
Purifying Corruption (4)
Medical Texts (2)
Archive of Conduits - Gateway to Paradise (4)
Meditative Trance (0)
Healing Words (3)
Bandages (0)
Medical Student (0)
Precious Memento - From a Former Life (4)
Precious Memento - From a Future Life (4)
Earthly Serenity (1)
Earthly Serenity (4)
Call for Backup (2)
Hallowed Chalice (0)
Lonnie Ritter - Feisty Mechanic (0)
```

```suggestion
                    "[Hh]eals? (that much )?((((\\d+)|(all)|(X total)) )?horror (from that asset )?(and|or) )?(((\\d+)|(all)|(X total)) )?damage"
```